### PR TITLE
Refac statedump

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Close https://github.com/neo-project/neo-modules/issues/736

### CODE REVIEW

File: src/StatesDumper/StatesDumper.cs [hunk 1]
```diff
@@ -122,45 +129,63 @@ private void OnPersistStorage(uint network, DataCache snapshot)
...
         {
             //Default Parameter
             uint storagePerFolder = 100000;
-            uint folder = (((blockIndex - 1) / storagePerFolder) + 1) * storagePerFolder;
-            if (blockIndex == 0)
-                folder = 0;
-            string dirPathWithBlock = $"./Storage_{network:x8}/BlockStorage_{folder}";
-            Directory.CreateDirectory(dirPathWithBlock);
-            return dirPathWithBlock;
+            uint folder = (blockIndex / storagePerFolder) * storagePerFolder;
```
Comment (roman-khimov) [inline src/StatesDumper/StatesDumper.cs:186]: This changes the naming scheme and will break scripts like https://github.com/nspcc-dev/neo-go/blob/master/scripts/compare-dumps/compare-dumps.go. Please keep the old scheme, I doubt changing it is important wrt to this PR goals.

Comment (Ashuaidehao) [reply src/StatesDumper/StatesDumper.cs:186]: Sure, I'll make a new simple plugin so you won't change it.

File: src/StatesDumper/Settings.cs [hunk 2]
```diff
@@ -26,10 +26,6 @@ internal class Settings
         /// </summary>
         public uint HeightToBegin { get; }
         /// <summary>
-        /// Height to begin real-time syncing and dumping on, consequently, dumping every block into a single files
-        /// </summary>
-        public int HeightToStartRealTimeSyncing { get; }
```
Comment (vncoelho) [inline src/StatesDumper/Settings.cs:31]: Why did you remove this configuration?
This can be useful when debugging or following the chain on real-time

**Comment by Ashuaidehao** [comment]
Replaced by  https://github.com/neo-project/neo-modules/pull/790
